build-system/tasks/storybook/index.js

```diff
@@ -36,6 +36,7 @@ function launchEnv(env) {
     [
       './node_modules/.bin/start-storybook',
       '--quiet',
+      '-s ../../../',
```

Comment:
accidental include?

Reply:
Intentional. I'm allowing our storybooks to access the static files we have in the project. For instance, one storybook here uses `iframe.html` static file.